BlitzBoard/Features/FixturesScreenViewController.swift

```diff
@@ -1,6 +1,69 @@
-//
 //  FixturesScreenViewController.swift
 //  BlitzBoard
 //
 //  Created by Kiasha Rangasamy on 2024/05/06.
 //
+
+import UIKit
+
+class FixturesScreenViewController: UIViewController {
+    
+    // MARK: IBOutlets
+    
+    @IBOutlet private weak var tableView: UITableView!
+    
+    // MARK: Variables
+    
+    var fixtures: [FixturesModel] = []
```

Comment:
Why do you have this property in your view controller? 
It should be in your viewModel since it is not UI related